068-performance-testing.md

```diff
@@ -0,0 +1,451 @@
+            ├── jvm_gc_pause_seconds_max{action="end of minor GC",cause="G1 Preventive Collection",gc="G1 Young Generation",}.txt
+            ├── jvm_memory_max_bytes.txt
+            ├── jvm_memory_used_bytes{area="heap",id="G1 Eden Space",}.txt
+            ├── jvm_memory_used_bytes{area="heap",id="G1 Old Gen",}.txt
+            ├── jvm_memory_used_bytes{area="heap",id="G1 Survivor Space",}.txt
+            ├── jvm_memory_used_bytes{area="nonheap",id="CodeHeap 'non-nmethods'",}.txt
+            ├── jvm_memory_used_bytes{area="nonheap",id="CodeHeap 'non-profiled nmethods'",}.txt
+            ├── jvm_memory_used_bytes{area="nonheap",id="CodeHeap 'profiled nmethods'",}.txt
+            ├── jvm_memory_used_bytes{area="nonheap",id="Compressed Class Space",}.txt
+            ├── jvm_memory_used_bytes{area="nonheap",id="Metaspace",}.txt
+            ├── jvm_memory_used_megabytes_total.txt
+            ├── jvm_threads_live_threads.txt
+            ├── process_cpu_usage.txt
+            ├── strimzi_add_finalizer_duration_seconds_max.txt
+            ├── strimzi_add_finalizer_duration_seconds_sum.txt
+            ├── strimzi_alter_configs_duration_seconds_max.txt
+            ├── strimzi_alter_configs_duration_seconds_sum.txt
+            ├── strimzi_create_partitions_duration_seconds_sum.txt
+            ├── strimzi_create_topics_duration_seconds_max.txt
+            ├── strimzi_create_topics_duration_seconds_sum.txt
+            ├── strimzi_delete_topics_duration_seconds_max.txt
+            ├── strimzi_delete_topics_duration_seconds_sum.txt
+            ├── strimzi_describe_configs_duration_seconds_max.txt
+            ├── strimzi_describe_configs_duration_seconds_sum.txt
+            ├── strimzi_describe_topics_duration_seconds_max.txt
+            ├── strimzi_describe_topics_duration_seconds_sum.txt
+            ├── strimzi_reconciliations_duration_seconds_max.txt
+            ├── strimzi_reconciliations_duration_seconds_sum.txt
+            ├── strimzi_reconciliations_failed_total.txt
+            ├── strimzi_reconciliations_locked_total.txt
+            ├── strimzi_reconciliations_max_batch_size.txt
+            ├── strimzi_reconciliations_max_queue_size.txt
+            ├── strimzi_reconciliations_successful_total.txt
+            ├── strimzi_reconciliations_total.txt
+            ├── strimzi_remove_finalizer_duration_seconds_max.txt
+            ├── strimzi_remove_finalizer_duration_seconds_sum.txt
+            ├── strimzi_resources.txt
+            ├── strimzi_total_time_spend_on_uto_event_queue_duration_seconds.txt
+            ├── strimzi_update_status_duration_seconds_max.txt
+            ├── strimzi_update_status_duration_seconds_sum.txt
+            ├── system_cpu_count.txt
+            ├── system_cpu_usage.txt
+            ├── system_load_average_1m.txt
+            ├── system_load_average_per_core_percent.txt
+            └── test-performance-metrics.txt
+... # more dates
```

Comment:
This can be shorter as well I think (I suppose that no-one will actually read it whole)

Reply:
yeah good idea